tests/Plugin_Main_Tests.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Tests for the Plugin_Main class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Plugin_Main;
+
+class Plugin_Main_Tests extends WP_UnitTestCase {
+	/**
+	 * Test class can be instantiated.
+	 *
+	 * @test
+	 */
+	public function it_can_be_instantiated() {
```

Comment:
See my comment on #44 for using the `test_` method prefix and avoiding doc-blocks (or more specifically the `@test` annotation) where not necessary.